### PR TITLE
Release August 2021 (1.0.0)

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
# oat-sa_extension-pcisample

## 3.1.0

 - Feature: make text reader pci to ims compliant

# oat-sa_extension-tao-backoffice

## 6.1.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-clientdiag

## 8.2.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-community

## 10.1.1

 - Fix: : update dependency to taoDeliveryRdf

# oat-sa_extension-tao-dac-simple

## 7.2.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-delivery-rdf

## 14.3.0

 - Feature: client-defined remotely-published delivery URI

# oat-sa_extension-tao-delivery

## 15.2.1

 - Fix: : move changes from testQti to taoDelivery extension

# oat-sa_extension-tao-eventlog

## 3.1.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-funcacl

## 7.0.1

 - ci: add automated release github action

# oat-sa_extension-tao-group

## 7.2.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-item

## 11.5.1

 - Fix: : allow user to move class when taoDacSimple is enabled.

# oat-sa_extension-tao-itemqti-pci

## 7.3.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-itemqti-pic

## 6.0.1

 - ci: add automated release github action

# oat-sa_extension-tao-itemqti

## 28.5.1

 - Fix: bump required lib version

# oat-sa_extension-tao-lti

## 12.1.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-ltideliveryprovider

## 12.1.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-mediamanager

## 12.6.2

 - Fix: : remove onoff sections toggling in passage authoring

# oat-sa_extension-tao-outcome

## 13.1.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-outcomekeyvalue

## 6.1.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-outcomelti

## 5.0.2

 - ci: add automated release github action

# oat-sa_extension-tao-outcomerds

## 8.1.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-outcomeui

## 10.6.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-proctoring

## 20.1.0

 - Feature: Updated Japanese translations

# oat-sa_extension-tao-revision

## 10.0.1

 - ci: add automated release github action

# oat-sa_extension-tao-task-queue

## 6.3.1

 - Feature: : fix return type

# oat-sa_extension-tao-test

## 15.1.1

 - Fix: : enable user to move class with tests

# oat-sa_extension-tao-testqti-previewer

## 3.1.0

 - Feature: Assign permissions to new item management roles

# oat-sa_extension-tao-testqti

## 41.10.1

 - Fix: test level timeout outcome

# oat-sa_extension-tao-testtaker

## 8.3.0

 - Feature: Revert "Added proper visible dependency from taoGroups extension"
 - Feature: Added proper visible dependency from taoGroups extension

# oat-sa_generis

## 15.0.0

 - Fix: : eliminate models table usage and legacy autoloader for the root-package installations
 - Merge branch 'r.3.0'
 - Feature: Add translations support for report

# oat-sa_tao-core

## 48.14.7

 - Fix: : update manifest with new sprint number

### CODE REVIEW

**Review by bziondik — APPROVED** [review]
LGTM

**Review by bziondik — COMMENTED** [review]
Maybe title should be `Release August 2021`?

**Review by clagosarias — APPROVED** [review]
LGTM, Great job!

**Review by habemuscode — APPROVED** [review]
Approved but should be done with `composer require extension-name:version -W` instead of `composer update -W`

With `composer update` also we update everything that can be updated, not only extensions, vendor too. And we have many recurrent issues with `ocramious/package` because there are in a PHP version not yet supported by us for example


@gabrielfs7

File: composer.json [hunk 1]
```diff
@@ -15,43 +15,46 @@
     }
   ],
   "keywords": [
-    "sprint 70",
+    "sprint 158",
```
Comment (pnal) [inline composer.json:18]:
```suggestion
    2021-08-lts",
```

**Review by pnal — APPROVED** [review]
LGTM

**Comment by bartlomiejmarszal** [comment]
> Approved but should be done with `composer require extension-name:version -W` instead of `composer update -W`
> 
> With `composer update` also we update everything that can be updated, not only extensions, vendor too. And we have many recurrent issues with `ocramious/package` because there are in a PHP version not yet supported by us for example
> 
> @gabrielfs7

I think this is a valid point. Nevertheless I had to use composer update as I did change `composer.json` content not related to `require` block.
This leads me to narrow php version to `composer.json` in a commit: https://github.com/oat-sa/tao-community/pull/259/commits/da20016d60568be3538ef0159497335911d577be

**Comment by pnal** [comment]
> > Approved but should be done with `composer require extension-name:version -W` instead of `composer update -W`
> > With `composer update` also we update everything that can be updated, not only extensions, vendor too. And we have many recurrent issues with `ocramious/package` because there are in a PHP version not yet supported by us for example
> > @gabrielfs7
> 
> I think this is a valid point. Nevertheless I had to use composer update as I did change `composer.json` content not related to `require` block.
> This led me to narrow php version to `composer.json` in a commit: [da20016](https://github.com/oat-sa/tao-community/commit/da20016d60568be3538ef0159497335911d577be)

May be I'm wrong but I see the safest way to release like:
1. copy both composer files from stabilization environment, cause there we have exact what was tested by QA: 
https://github.com/oat-sa/deploy-test-package/blob/tao/community-202108-lts/files/delivery/composer.json
2. change `preferred install` to `dist`
3. add `preferred stable: true` 
4. run `composer install`
5. run `composer require package:version -w` (uppercase -W only if that not working due to some conflicts) for all the packages which needed to be changed to appropriate version

**Review by habemuscode — APPROVED** [review]

**Review by bziondik — APPROVED** [review]
LGTM

**Review by boajer — APPROVED** [review]

**Pull request merged**